client/src/pages/Profile.jsx

```diff
@@ -1,14 +1,59 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
+import { getAttendingSessions } from '../utils/apiWrapper';
+import 'semantic-ui-css/semantic.min.css';
+import '../css/Profile.scss';
+import SessionSummary from '../components/SessionSummary';
+import ClassForm from '../components/ClassForm.jsx';
 import ClassCard from '../components/ClassCard';
 
 function Profile({ user }) {
```

Comment:
Please pass in just `props` into Profile, and then deconstruct the props on the first line like `const { user } = props;`. This is because passing in just `user` doesn't allow the Profile component to take in other props (you're hard-coding it) while passing in `props` allows as many props as we want.